packages/zoe/src/internal-types.js

```diff
@@ -96,14 +96,17 @@
  * @typedef {Object} InstanceAdmin
  * @property {() => void} assertAcceptingOffers
  * @property {(invitationHandle: InvitationHandle,
-      initialAllocation: Allocation,
-      proposal: ProposalRecord) => UserSeat } makeUserSeat
+ *     initialAllocation: Allocation,
+ *     proposal: ProposalRecord) => Promise<UserSeat> } makeUserSeat
```

Comment:
The type here was correct, but the code has an unnecessary `async`. We should remove that instead.

Reply:
I don't know how to.  Can you please help?